docs/installation/bootstrap.rst

```diff
@@ -130,6 +134,13 @@ The ``archives`` field is a list of absolute paths to MetalK8s ISO files. When
 the bootstrap script is executed, those ISOs are automatically mounted and the
 system is configured to re-mount them automatically after a reboot.
 
+The ``kubernetes`` field can be omitted if you do not have any specific
+Kubernetes `Feature Gates`_ to enable or disable.
+
+    If you want to enable or disable a specific feature for ``kube-apiserver``
```

Comment:
Why is this indented?

Reply:
Just to add several paragraphs for various "kubernetes" components in the future

Reply:
Maybe should remove this indentation here since we will release this version with only apiserver

Reply:
I removed indentation here